tools/rosgraph/src/rosgraph/xmlrpc.py

```diff
@@ -129,14 +130,18 @@ def handle_error(self, request, client_address):
             logger = logging.getLogger('xmlrpc')
             if logger:
                 logger.error(traceback.format_exc())
-    
-class ForkingXMLRPCServer(socketserver.ForkingMixIn, SimpleXMLRPCServer):
-    """
-    Adds ThreadingMixin to SimpleXMLRPCServer to support multiple concurrent
-    requests via forking. Also makes logging toggleable.      
-    """
-    def __init__(self, addr, request_handler=SilenceableXMLRPCRequestHandler, log_requests=1):
-        SimpleXMLRPCServer.__init__(self, addr, request_handler, log_requests)
+
+
+# ForkingMixIn and the Forking classes mentioned below are only available on POSIX 
+# platforms that support fork().
+if hasattr(os, "fork"):
```

Comment:
Rather than import `os` for this check, what about just `hasattr(socketserver, 'ForkingMixIn')`?

Reply:
@mikepurvis That works too. I simplified the check and rebased the PR to the latest.